backtrace-library/src/main/java/backtraceio/library/base/BacktraceBase.java

```diff
@@ -363,94 +378,107 @@ public boolean clearBreadcrumbs() {
-     * @return              true if the breadcrumb was successfully added
+     *
+     * @param message a message which describes this breadcrumb (1KB max)
+     * @param type    broadly describes the category of this breadcrumb
+     * @param level   the severity level of this breadcrumb
+     * @return true if the breadcrumb was successfully added
      */
     public boolean addBreadcrumb(String message, BacktraceBreadcrumbType type, BacktraceBreadcrumbLevel level) {
         return database.getBreadcrumbs().addBreadcrumb(message, type, level);
     }
 
     /**
      * Add a breadcrumb of the desired type and level "Info" with the provided message string and attributes
-     * @param message       a message which describes this breadcrumb (1KB max)
-     * @param attributes    key-value pairs to provide additional information about this breadcrumb (1KB max, including some overhead per key-value pair)
-     * @param type          broadly describes the category of this breadcrumb
-     * @return              true if the breadcrumb was successfully added
+     *
+     * @param message    a message which describes this breadcrumb (1KB max)
+     * @param attributes key-value pairs to provide additional information about this breadcrumb (1KB max, including some overhead per key-value pair)
+     * @param type       broadly describes the category of this breadcrumb
+     * @return true if the breadcrumb was successfully added
      */
     public boolean addBreadcrumb(String message, Map<String, Object> attributes, BacktraceBreadcrumbType type) {
         return database.getBreadcrumbs().addBreadcrumb(message, attributes, type);
     }
 
     /**
      * Add a breadcrumb of the desired level and type with the provided message string and attributes
-     * @param message       a message which describes this breadcrumb (1KB max)
-     * @param attributes    key-value pairs to provide additional information about this breadcrumb (1KB max, including some overhead per key-value pair)
-     * @param type          broadly describes the category of this breadcrumb
-     * @param level         the severity level of this breadcrumb
-     * @return              true if the breadcrumb was successfully added
+     *
+     * @param message    a message which describes this breadcrumb (1KB max)
+     * @param attributes key-value pairs to provide additional information about this breadcrumb (1KB max, including some overhead per key-value pair)
+     * @param type       broadly describes the category of this breadcrumb
+     * @param level      the severity level of this breadcrumb
+     * @return true if the breadcrumb was successfully added
      */
     public boolean addBreadcrumb(String message, Map<String, Object> attributes, BacktraceBreadcrumbType type, BacktraceBreadcrumbLevel level) {
         return database.getBreadcrumbs().addBreadcrumb(message, attributes, type, level);
     }
 
+    public void setCurrentBreadcrumbId(long breadcrumbId) {
```

Comment:
I would avoid exposing this API from baktraceclient - why any user would like to use it?

Reply:
I think we can avoid adding this here, thank you!